### ISSUE TITLE
Problem with linking to locally downloaded assessment items on Debian install

### ISSUE THREAD

**Issue by radinamatic** [learningequality/installers#356]
Problem with linking to locally downloaded assessment items on Debian install

##### Build/Version

0.15 (build `ka-lite_0.15.0-0ubuntu4_all.deb`) on Ubuntu 15.10

It's not worth correcting this for 0.15, but I'm filing this issue just so we make sure that the same does not occur on 0.16 .deb installer that is yet to be compiled. 

##### Expected Behavior

If user uses the `ka-lite_0.15.0-0ubuntu4_all.deb` installer and has downloaded the `khan_assessment.zip` file, they should be able to link to that file during installation process.

##### Current Behavior

At select the .zip file step:

![path](https://cloud.githubusercontent.com/assets/1457929/12784000/e0dfa644-ca84-11e5-93b9-25a64e63eedb.png)

The recommended phrase/path should be improved for clarity. My first try was to leave just the file name (instead of the link that comes up by default) since the Terminal was running the installer in the same home/Downloads folder where the the .zip file was located. 

```
--2016-02-03 14:07:59--  (try: 6)  http://khan_assessment.zip/
Connecting to khan_assessment.zip (khan_assessment.zip)|127.0.53.53|:80... failed: Connection refused.
Retrying.

--2016-02-03 14:08:00--  (try: 7)  http://khan_assessment.zip/
Connecting to khan_assessment.zip (khan_assessment.zip)|127.0.53.53|:80... failed: Connection refused.
Retrying.

^C
dpkg: error processing package ka-lite (--install):
 subprocess installed post-installation script was interrupted
Processing triggers for man-db (2.7.4-1) ...
Errors were encountered while processing:
 ka-lite
didi@didi-VirtualBox:~/Downloads$
``` 

After that I tried `file:///khan_assessment.zip`, and `home/didi/Downloads/khan_assessment.zip` with none, 1-3 slashes after `file`, and kept getting variations of `wget: unable to resolve host address ‘file’` error:

```
didi@didi-VirtualBox:~/Downloads$ sudo dpkg -i ka-lite_0.15.0-0ubuntu4_all.deb
[sudo] password for didi:
Selecting previously unselected package ka-lite.
(Reading database ... 239027 files and directories currently installed.)
Preparing to unpack ka-lite_0.15.0-0ubuntu4_all.deb ...
Unpacking ka-lite (0.15.0-0ubuntu4) ...
Setting up ka-lite (0.15.0-0ubuntu4) ...
--2016-02-03 15:17:33--  ftp://file//home/didi/Downloads/khan_assessment.zip
           => ‘/tmp/assessment_items.zip’
Resolving file (file)... failed: Temporary failure in name resolution.
wget: unable to resolve host address ‘file’
dpkg: error processing package ka-lite (--install):
 subprocess installed post-installation script returned error exit status 4
Processing triggers for man-db (2.7.4-1) ...
Errors were encountered while processing:
 ka-lite
didi@didi-VirtualBox:~/Downloads$
```

##### NOTE
If I restart the installer, skip the items step and choose to do that at `kalite manage setup`, it works without problems.

**Comment by benjaoming** [comment]
@radinamatic `file://` denotes the protocol, the third slash `/` denotes an absolute path.

**Comment by benjaoming** [comment]
@radinamatic it's a fundamental part of the current automated tests to be able to use local filesystem resources for the assessment zips, so I can't imagine it'd be broken :)

**Comment by benjaoming** [comment]
Fixed in #367
